website/data/docs-nav-data.json

```diff
@@ -1310,6 +1310,11 @@
             "title": "Considerations",
             "path": "secrets/pki/considerations"
           },
+          {
+            "title": "Troubleshooting - ACME",
```

Comment:
Suggest simplifying the nav title to "Troubleshoot ACME integration" or something similar that provides more information about the specific type of problems the troubleshooting guide will help with.

Reply:
I think this should be consistent with the heading, got a preference between `Troubleshoot ACME Integration` or `Troubleshoot PKI Secrets Engine and ACME`?

Reply:
Nav entries should be similar to the full document title, but there's no requirement that the nav entry and the document title be *exactly* the same. It's often useful for the page title be very specific about the concept/procedure/etc covered (both for reader clarity and improved SEO), which can result in long-ish titles that don't work well in the nav.  

The reason I recommended "Troubleshoot ACME integration" for the nav is because the "PKI Secrets Engine" part is already implied by the fact that the doc is listed under that section in the navigation. On the other hand, using the broader title "Troubleshoot PKI Secrets Engine and ACME [integration]" in the actual document helps provide the full context for readers who may end up on the doc via a web search and won't necessarily notice where it appears in the nav structure.